src/App.tsx

```diff
@@ -1,156 +1,72 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Sort } from './utils/enums';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+const USER_ID = 10284;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sort, setSort] = useState(Sort.All);
```

Comment:
The name `sort` is a bit confusing in this case, because sorting is more about changing the order of items, and what you're actually doing is filtering.
```suggestion
  const [filter, setFilter] = useState(FilterType.All);
```